src/abstractdataframe/sort.jl

```diff
@@ -397,14 +398,20 @@ function Base.issorted(df::AbstractDataFrame, cols=All();
                        lt::Union{Function, AbstractVector{<:Function}}=isless,
                        by::Union{Function, AbstractVector{<:Function}}=identity,
                        rev::Union{Bool, AbstractVector{Bool}}=false,
-                       order::Union{Ordering, AbstractVector{<:Ordering}}=Forward)
+                       order::Union{Ordering, AbstractVector{<:Ordering}}=Forward,
+                       checkunique::Bool=false)
     to_scalar(x::AbstractVector) = only(x)
     to_scalar(x::Any) = x
 
     # exclude AbstractVector as in that case cols can contain order(...) clauses
     if cols isa MultiColumnIndex && !(cols isa AbstractVector)
         cols = index(df)[cols]
     end
+    if checkunique
+        if !allunique(df, cols)
+            error("Non-unique elements found. Multiple orders are valid.")
```

Comment:
`ArgumentError` should be thrown.

Reply:
Is the error message clear enough?

Reply:
It is for me. Usually @nalimilan has more detailed comments relatad with messages.